README.md

```diff
@@ -1,131 +1,236 @@
+  This option will accept aruguments and return or save some kind of logs.
+    
+  ```
+  ae logs <options>
+  ```
+    
+</details>
+
+<details>
+  <summary><code>oci</code></summary>
+  
+  &nbsp;
+  
+  Here the [OCI CLI interface](https://github.com/opencontainers/runtime-tools/blob/master/docs/command-line-interface.md) is implemented with the respective subcommands.
+    
+  ```
+  ae oci
+  ae oci create
+  ae oci delete
+  ae oci kill
+  ae oci start
+  ae oci status
+  ```
+    
+</details>
+
+<details>
+  <summary><code>start</code></summary>
+  
+  &nbsp;
+  
+  It will run the rescource directly.
+    
+  ```
+  ae start
+  ae start executable, exe
+  ae start container # Note this has an alias: 'ae oci start'
+  ```
+    
+</details>
+
+<details>
+  <summary><code>stop</code></summary>
+  
+  &nbsp;
+  
+  It will stop the rescource directly.
+    
+  ```
+  ae stop
+  ae stop executable, exe
+  ae stop container # Note this has an alias: 'ae oci kill'
+  ```
+    
+</details>
+
+<!-- PHILOSOPHY -->
+## Philosophy
+    
+### Less is more
+    
 We do not want `ae` to become a junk drawer. In situations where we are considering bringing in new functionality to the project, we prefer to keep it out.
 
-For example imagine we were considering bringing in a `--filter`flag or a `--query` flag which would allow for us to filter the output returned by `ae`. In this situation we could very well identify advanced query patterns and libraries that would make it possible to adopt this feature.
+For example imagine we were considering bringing in a `--filter` flag or a `--query` flag which would allow for us to filter the output returned by `ae`. In this situation we could very well identify advanced query patterns and libraries that would make it possible to adopt this feature.
 
 However the feature comes at a maintenance cost, and frankly already exists in other tools. Instead of adopting a filter syntax and the associated burden that comes along with it we instead focus on outputting to formats (such as JSON) which can easily be queried by existing tools (such as [jq](https://stedolan.github.io/jq/)).
-
+    
 ### Just because you can, doesn't mean you should.
-
+    
 Also known as **"The Jurassic Park"** rule.
 
 In situations where functionality is made available or "is possible" because of current technical advancements or capabilities, we prefer to focus on the goals of ae instead of supporting optional features "just because we can".
 
 For example imagine discovering a well written, and well regarded library that takes a Go struct and turns it into many formats which can be printed to the screen such as XML, JSON, YAML, TOML, BSON, and so on....
 
 The project currently has established a need for JSON only, and plans to use jq for filtering and querying the data. In this case bringing the additional output types to the project "just because we can" would be a violation of this rule.
-
-### No assumptions
+    
+### No assumptions…
```

Comment:
nit: trailing _

Reply:
solved by 2c5ec4f. actually wrote it because i thought it fitted, but this is indeed better!